README.md

```diff
@@ -1,36 +1,25 @@
-# Address Book (Level 4)
+# main
 
-[![Build Status](https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
-[![Build status](https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true)](https://ci.appveyor.com/project/damithc/addressbook-level4)
-[![Coverage Status](https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master)](https://coveralls.io/github/se-edu/addressbook-level4?branch=master)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a)](https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=se-edu/addressbook-level4&amp;utm_campaign=Badge_Grade)
+[![Build Status](https://travis-ci.org/CS2103JAN2017-F11-B2/main.svg?branch=master)](https://travis-ci.org/CS2103JAN2017-F11-B2/main)
+[![Build status](https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true)](https://ci.appveyor.com/project/damithc/main)
+[![Coverage Status](https://coveralls.io/repos/github/CS2103JAN2017-F11-B2/main/badge.svg?branch=master)](https://coveralls.io/github/CS2103JAN2017-F11-B2/main?branch=master)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a)](https://www.codacy.com/app/damith/main?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=CS2103JAN2017-F11-B2/main&amp;utm_campaign=Badge_Grade)
```

Comment:
This is not the correct update.
Refer to the Codacy badge for more information.